packages/edit-site/src/components/header/index.js

```diff
@@ -79,6 +90,8 @@ export default function Header( {
 			listViewShortcut: getShortcutRepresentation(
 				'core/edit-site/toggle-list-view'
 			),
+			editorMode: getEditorMode(),
```

Comment:
I'm interested in whether it makes sense to conceptualize this as a "zoom" level that could range from focusing on a content to template to full template to mosaic.

Reply:
Hi @mtias, to me it seems a zoom concept makes sense. For the mosaic view, it seems a good fit but we would need to know how the zoom is going to be used for example to focus the content.
On the side editor, we can edit the content of a specific post but it seems we can not focus on the content and we always see the full site. Do you think we should implement a feature to focus the content of a post on the site editor (making it look almost equal to the post editor)?